src/utils/sort.deadlines.js

```diff
@@ -0,0 +1,55 @@
+module.exports = (query, data) => {
+	const supportedQueryParams = [
+		'duedate-asc',
+		'duedate-desc',
+		'newest-to-oldest',
+		'oldest-to-newest',
+	]
+	if (!supportedQueryParams.includes(query)) return data
+
+	if (query.toLowerCase() === 'duedate-asc') {
+		data.sort((b, a) => {
+			// console.log(a, b)
+			const result =
+				new Date(b.dueDate).getTime() - new Date(a.dueDate).getTime()
+			console.log(result)

```

Comment:
you need to remove the console log

everything looks good, I love the fact that it doesn't throw error if anything goes wrong and just return the data as it was. Good job Bro!!

Reply:
Thanks man